src/platform/qt/SettingsView.cpp

```diff
@@ -577,9 +577,9 @@ void SettingsView::reloadConfig() {
 	loadSetting("lockAspectRatio", m_ui.lockAspectRatio);
 	loadSetting("lockIntegerScaling", m_ui.lockIntegerScaling);
 	loadSetting("interframeBlending", m_ui.interframeBlending);
-	loadSetting("showOSD", m_ui.showOSD, true);
```

Comment:
This should not be removed; it's for setting the default when no option is present and entirely unrelated to the bug in question. Why did you remove it?